ocs_ci/ocs/resources/objectconfigfile.py

```diff
@@ -135,3 +136,57 @@ def get(self, namespace=None):
         """
         out = self._run_command("get", namespace, out_yaml_format=True)
         return yaml.safe_load(out)
+
+    def describe(self, namespace=None):
+        """
+        Run ``oc describe`` on in this object file.
+
+        Args:
+            namespace (str): Name of the namespace where to deploy, overriding
+            self.project.namespace value (in a similar way how you can specify
+            any value to ``-n`` option of ``oc describe``.
+        """
+        out = self._run_command("describe", namespace, out_yaml_format=False)
+        return yaml.safe_load(out)
+
+    def wait_for_delete(self, resource_name="", timeout=60, sleep=3, namespace=None):
```

Comment:
Since the current implementation is waiting for namespace deletion, I'd call it "wait_for_namespace_deletion" or anything else which will reflect the fact that we delete a namespace and not a single resource inside it.

Reply:
It's not namespace deletion, Namespace is created using factory_fixture and deletion will be handled part of it. This function is for validating either respective kube_job on the specified namespace is delete or not.
This is generic function for any kube_job obj, i.e. it can be used for POD/PVC/Clone/Snapshot kube_job obj, you just need to provide the resource name(kube_job_obj) to validate.